CHANGELOG.md

```diff
@@ -1,5 +1,94 @@
 # Changelog
 
+## v1.8.7
+
+Summary of Changes
+------------------
+
+**Minor Changes:**
+* [v1.8] k8s: Update libraries to v1.18.13 (#14408, @christarazi)
+* [v1.8] k8s: Update libraries to v1.18.14 (#14479, @christarazi)
+* daemon, node: refresh neighbor by sending arping periodically (Backport PR #14574, Upstream PR #14498, @jaffcheng)
+* ipsec: Fatal on unsupported, <4.19 kernels in tunneling mode (Backport PR #14586, Upstream PR #14525, @pchaigno)
+* Make Go runtime to return unused memory to OS more often (Backport PR #14953, Upstream PR #14634, @aanm)
+
+**Bugfixes:**
+* cilium-cni: Fix error handling for bad netns (Backport PR #14654, Upstream PR #14645, @joestringer)
+* clustermesh: Ignore symlink files on fsnotify events (Backport PR #14654, Upstream PR #14565, @tgraf)
+* Fix a bug that affects connectivity to NodePort service via ExternalIP of the local k8s node. (Backport PR #14953, Upstream PR #14793, @AnishShah)
+* Fix a route MTU issue where pods cannot receive large packets from outside the cluster 
 when the sender sets the "don't fragment" (DF) bit. (Backport PR #14740, Upstream PR #14679, @aditighag)
```

Comment:
Not sure where the ^M comes from, but maybe better to remove. It doesn't seem to have any effect though.